.github/workflows/build-apps.yml

```diff
@@ -362,6 +362,17 @@ jobs:
       - name: List artifacts
         run: "ls -R out"
 
+      - name: Set more complete nightly release notes
+        if: ${{ env.IS_NIGHTLY == 'true' }}
+        run: |
+          # Note: prefered going this way instead of a full clone in the checkout step,
```

Comment:
```suggestion
          # Note: preferred going this way instead of a full clone in the checkout step,
```

Reply:
Crap with the auto merge thing it went in before I could fix the typo. Will do next time I edit this file

Reply:
Fixed in #4627